cmd/loop/loopout.go

```diff
@@ -87,6 +89,17 @@ func loopOut(ctx *cli.Context) error {
 		return err
 	}
 
+	// Parse outgoing channel set.
+	chanStrings := strings.Split(ctx.String("channel"), ",")
+	var outgoingChanSet []uint64
+	for _, chanString := range chanStrings {
```

Comment:
Is there any need to check that the set of channels provided are unique? I doubt it, but wanted to check that it won't behave strangely with duplicates on the lnd side.

Reply:
It is checked in `loopd`

Reply:
I don't know how seriously we account for the alternate client case?

Reply:
The alternate client would still connect to `loopd`?